deploy/kubernetes/scripts/init-helm.sh

```diff
@@ -3,13 +3,14 @@
 TILLER_DEPLOY=`kubectl get deploy -n kube-system | grep 'tiller-deploy'`
 [ -n "$TILLER_DEPLOY" ] && echo "Helm is already initialized." && exit 1
 
+export DESIRED_VERSION=v2.12.0
 
-curl https://raw.githubusercontent.com/helm/helm/master/scripts/get | DESIRED_VERSION=v2.12.0 bash
+curl https://raw.githubusercontent.com/helm/helm/${DESIRED_VERSION}/scripts/get | bash
 
-helm init --tiller-image=gcr.io/kubernetes-helm/tiller:v2.12.0 --upgrade
+helm init --tiller-image=gcr.io/kubernetes-helm/tiller:${DESIRED_VERSION}
```

Comment:
If tiller already exist, need --upgrade

Reply:
This script is used for travis, so the tiller not exist; But there is no harm if add it, I would add it.